tests/vmi_lifecycle_test.go

```diff
@@ -80,6 +80,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 	var allowEmulation *bool
 
 	const fakeLibvirtLogFilters = "3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 1:*"
+	const startupTimeout = 45
```

Comment:
This would slow down (failing) tests for x86, too. Would it make sense to have this conditioned on the architecture?

Reply:
Wouldn't it be too complicated? We do not have unified timeouts over the whole test suite yet...

Reply:
Maybe it's complicated, but I think that we are too quick to increase timeouts, while our tests are already way too long. If I look on this test file, are all these test even required?  We run VMs in hundreds of other places do we need another racy one? Why do we need one test to create a VMI, another to run it, and another to check that virt-launcher exists?

Reply:
I think that the point here is that @zhlhahaha sees this in this test file just because it uses particularly *short* timeouts. Elsewhere in our codebase we have `libwait.defaultTimeout = 360`.

Come to think of it, I guess that the slow startup time for ARM may be related to Arm as an architecture, but to the specific Arm CI which is much smaller than the x86 one. So should not make the timeout Arch-dependent.

We should still delete/merge some of the superfluous tests in this file, though.

Reply:
>Maybe it's complicated, but I think that we are too quick to increase timeouts, while our tests are already way too long. If I look on this test file, are all these test even required? 

No, I am working on it, I have something locally already. I try to push it today.

>We run VMs in hundreds of other places do we need another racy one? Why do we need one test to create a VMI, another to run it, and another to check that virt-launcher exists?

+1 This is something we can effort in unit tests but e2e seems unresonable.

>It is not complicated.
The same way we initialize libvmi with the arch, a variable can be initialized for use in this test file or globally. Instead of a specific timeout, consider using an offset (e.g. startupTime + archOffsetTime).

Sure, but is it really required? I am not sure.
What I take from this is that we & maybe @zhlhahaha will need to have a look at the difference...

>Come to think of it, I guess that the slow startup time for ARM may be related to Arm as an architecture, but to the specific Arm CI which is much smaller than the x86 one. So should not make the timeout Arch-dependent.

And this is exactly why I want to merge this and investigate after. 

>We should still delete/merge some of the superfluous tests in this file, though.

+1 but that should not block this or any other PR

Reply:
Hi, @xpivarc @dankenigsberg @EdDev , thanks for reviewing the PR.
> Arch specific timeout

While it's not complicated, I don't believe it's necessary. The difference in timeout only occurs in the VMI lifecycle-related tests, and since we're not significantly extending the timeout, it shouldn't greatly impact the overall e2e tests. However, investigating the startup time discrepancy between the Arm64 and x86_64 platforms is indeed worthwhile.

> redundancy of E2E tests

I think it's worthwhile to review the necessity of the E2E tests.

Reply:
> I think it's worthwhile to review the necessity of the E2E tests

Looking forward to seeing a PR removing a redundant test, @zhlhahaha !